src/pages/users/management/UserSectionPills.module.scss

```
@@ -0,0 +1,167 @@
+    }
+
+    &:focus-visible {
+        outline: 2px solid #141c25;
+        outline-offset: 2px;
+    }
+}
+
+.pillIcon {
+    flex-shrink: 0;
+    font-size: 16px;
+    color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)